x/coinswap/types/msgs.go

```diff
@@ -23,12 +23,16 @@ const (
 	LptTokenFormat = "lpt-%d"
 )
 
+/* --------------------------------------------------------------------------- */
+// MsgSwapOrder
+/* --------------------------------------------------------------------------- */
+
 // MsgSwapOrder - struct for swapping a coin
 // Input and Output can either be exact or calculated.
 // An exact coin has the senders desired buy or sell amount.
 // A calculated coin has the desired denomination and bounded amount
 // the sender is willing to buy or sell in this order.
-//
+
```

Comment:
```suggestion
//
```

if you need new line for comments, please use empty comment rather than empty line